docs/internal/investigations/remote-runtime-configuration-dapr/poc/dapr-poc-scenario.md

```diff
@@ -0,0 +1,143 @@
+- helm 3 [https://helm.sh/](https://helm.sh/)
+- watch (`brew install watch`)
+
+Ensure you have all the tools installed and start in the `poc` directory.
+
+> NOTE: Please keep in mind that some shells automatically escape some characters when copying and pasting commands. Always check the command before running it.
+
+### Control Plane deployment
+
+In the first step, we deploy the Control Plane cluster which hosts the Runtime Governor component as well as two Redis instances for the purpose of persisting the state by the Runtimes.
+
+To deploy the Control Plane cluster run the following command.
+
+```
+$ ./setup-cp.sh
+```
+
+You can watch the installation progress by looking at the statuses of the pods being deployed in the cluster.
+
+```
+$ watch kubectl --context=k3d-cp get po --all-namespaces
+```
+
+### Runtimes deployment
+
+The next step is to deploy two runtimes that will consume the Control Plane API and persist the state in the Redis instances configured by the central Control Plane.
+
+To deploy the first Runtime cluster run the following command. It will create the cluster and deploy a basic set of components including the NodeApp that demo the use-case with the state store.
+
+```
+$ ./setup-runtime1.sh
+```
+
+You can watch the installation progress by looking at the statuses of the pods being deployed in the cluster. On that stage, there should be NodeApp Pod deployed but the Dapr sidecar should not be injected yet.
+
+```
+$ watch kubectl --context=k3d-runtime1 get po --all-namespaces
+```
+
+Next, deploy the agent component using the following command. It will deploy the agent that connects to the Control Plane cluster and fetches the configuration for the given runtime.
+
+```
+$ ./setup-runtime1-agent.sh
+```
+
+You can watch the installation progress by looking at the statuses of the pods being deployed in the cluster. After a successful deployment of the agent component, it should fetch the configuration and apply it to the cluster. The NodeApp pod should be recreated with the Dapr sidecar injected.
+
+```
+$ watch kubectl --context=k3d-runtime1 get po --all-namespaces
+```
+
+In order to run the second runtime cluster please run all the above steps using `runtime2` scripts.
+
+### Use-case scenario
+
+The setup described gives you the possibility to interact with two runtime clusters. On both runtimes, there is an example Dapr NodeApp deployed. It exposes two endpoints `/order`, and `/neworder` which interacts with the Redis state store. You can call those endpoints using the docker container run in the same docker network as the clusters are running. 
+
+You can call `/order` endpoint of the runtime1 and runtime2, running the following commands:
+
+```
+// Runtime 1
+$ docker run --network=cpnet --rm --entrypoint curl curlimages/curl -iv http://${$(docker inspect k3d-runtime1-server-0 --format='{{json .NetworkSettings.Networks.cpnet.IPAddress}}')//\"}/order
```

Comment:
I think that according to the guidelines the `$` sign at the beginning should be removed

Reply:
That's a pity, but I'll remove it.